web/themes/joinup/joinup_theme.theme

```diff
@@ -89,11 +103,27 @@ function joinup_theme_preprocess_page(&$variables) {
 
     if (in_array($rdf_type, ['collection', 'solution'])) {
       $variables['has_vertical_tabs'] = TRUE;
+      $is_form_container = TRUE;
+    }
+  }
+
+  // Check pattern for add pages.
+  $form_pattern = '.add';
```

Comment:
Ok, I moved two checks to a function:
* if page is a create page
* if page has vertical tabs
There is a possibility to extend the function and add new checks in the future.